internal/consensus/reactor.go

```diff
@@ -1208,6 +1175,66 @@ func (ps *PeerState) setHasProposalBlockPart(height int64, round int32, index in
 	ps.PRS.ProposalBlockParts.SetIndex(index, true)
 }
 
+// SendPartSetHasPart sends the part to the peer.
+// Returns true and marks the peer as having the part if the part was sent.
+func (ps *PeerState) SendPartSetHasPart(part *types.Part, prs *cstypes.PeerRoundState) bool {
+	// Send the part
+	ps.logger.Debug("Sending block part", "height", prs.Height, "round", prs.Round, "index", part.Index)
+	pp, err := part.ToProto()
+	if err != nil {
+		// NOTE: only returns error if part is nil, which it should never be by here
```

Comment:
This was originally a panic. Are we ok dropping it in favor of a logger.Error() ?